config/initializers/apipie.rb

```diff
@@ -99,8 +99,6 @@
   else
     warn "Apipie cache enabled but not present yet. Run apipie:cache rake task to speed up API calls."
   end
```

Comment:
Do we even need this whole block since we hard coded the setting to not use the cache?

Reply:
Since someone could come turn it on per your idea -- https://github.com/theforeman/foreman/pull/9165#discussion_r855075385 I would argue to keep it?